### ISSUE TITLE
Don't run as root

### ISSUE THREAD

**Issue by mvdan** [androportal/f-droid-fdroidserver#1]
Hello androportal,

I see you wrote a guide that's quite different to our manual at f-droid.org/manual. That's completely fine. The only issue I see with it is that you advise people to place fdroid under /usr/bin and run it as root.

Placing it under /usr/bin isn't that bad, just a bit messy. You could play with PATH or aliases, it would be much cleaner. On the other hand, advising to run fdroid as root is a very bad idea. It doesn't need root, and running build recipes as root is very dangerous (for exaple, in a recipe someone could put `init=rm -rf /`).

Your fork hasn't been updated in a while, which also leads to people reporting issues to us that were solved months ago. Could you please link to our repo and/or update your fork?

Thank you!

**Comment by psachin** [comment]
Hello Daniel,

Thanx for raising an issue. I have made changes as per your suggestions. Below is the list of changes: 
- Added official fdroid repo address.
- Instruction to use PATH instead of copying 'fdroid' binary to '/usr/bin'.
- Revoked root execution of fdroid binary.
- Added official fdroid manual link.

As the forked repository is best suited to our need we are not planning to update it for a time being, instead we have included link to your repo. Also we have provided link to original fdroid manual for future reference to users.

Hope this solves the issue.
